tests/api_connexion/endpoints/test_dag_endpoint.py

```diff
@@ -110,22 +115,21 @@ def test_should_respond_200(self):
         self._create_dag_models(1)
         response = self.client.get("/api/v1/dags/TEST_DAG_1", environ_overrides={'REMOTE_USER': "test"})
         assert response.status_code == 200
-
-        current_response = response.json
-        current_response["fileloc"] = "/tmp/test-dag.py"
+        file_token = SERIALIZER.dumps("/tmp/dag_1.py")
```

Comment:
This is going to be constant as long as the secret key is constant, right? (And that is defined in our test config). So could we include the actual result once in one test, to ensure that it is actually encoding it correctly.

Reply:
Ok. Working on it